# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: toolkit/components/cascade_bloom_filter/Cargo.toml
# Commit: b2e8af2fabb0
# Full Hash: b2e8af2fabb053ac02070415e9014a8a5e356d31
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2022-03-10 17:23:56
# Description:
#   Bug 1615974 - avoid memmapping CRLite filters in cert_storage r=jschanck,robwu
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D140266
# ==============================================================================

diff -r 902ab4b1756d -r b2e8af2fabb0 toolkit/components/cascade_bloom_filter/Cargo.toml
--- a/toolkit/components/cascade_bloom_filter/Cargo.toml	Wed Mar 09 22:35:35 2022 +0000
+++ b/toolkit/components/cascade_bloom_filter/Cargo.toml	Wed Mar 09 22:46:15 2022 +0000
@@ -6,7 +6,6 @@
 [dependencies]
 nserror = { path = "../../../xpcom/rust/nserror" }
 nsstring = { path = "../../../xpcom/rust/nsstring" }
-rental = "0.5.5"
-rust_cascade = "0.6.0"
+rust_cascade = "1.2.0"
 thin-vec = { version = "0.2.1", features = ["gecko-ffi"] }
 xpcom = { path = "../../../xpcom/rust/xpcom" }